test/include/postprocessors/TestDiscontinuousValuePP.h

```diff
@@ -0,0 +1,80 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+#ifndef TESTDISCONTINUOUSVALUEPP_H
+#define TESTDISCONTINUOUSVALUEPP_H
+
+#include "GeneralPostprocessor.h"
+
+//Forward Declarations
+class TestDiscontinuousValuePP;
+class SolutionUserObject;
+
+template<>
+InputParameters validParams<TestDiscontinuousValuePP>();
+
+/**
+ * Compute the value of a variable or the gradient at a specified location.
+ * Takes multivalued functions at boundaries into account
+ */
+class TestDiscontinuousValuePP : public GeneralPostprocessor
+{
+public:
+  /**
+   * Constructor.
+   * @param parameters The input file parameters for this object
+   */
+  TestDiscontinuousValuePP(const InputParameters & parameters);
+
+  /**
+   * Destructor
+   */
+  virtual ~TestDiscontinuousValuePP(){};
+
+  /// initialize override
```

Comment:
The style for methods IIRC is:
```
/**
 *
 */
```

And again, doco strings for the sake of doco strings...